gla_blockquote.features.yml

```diff
@@ -0,0 +1 @@
+required: true
```

Comment:
_Features_?! In Drupal 9?! After Luke did all that work to look at Config Updates and ruled out Features?!

I'm not going mad – Features is actually great for maintainers of profiles/distributions to use _locally as a development-only tool_ to package up related configuration, automatically move it into a module's `config` directory, and to apply transformations like removing site UUIDs and hashes, as well as adding dependencies. It beats manually copying things out of `config/sync` by a thousand. I'll do a demo next week if you like?

Reply:
https://www.drupal.org/docs/contributed-modules/features/whats-new-in-features-for-drupal-8 has a bit more info, the key points are these:

> Purely a building tool. Previous versions of Features included extensive workarounds to make up for the lack of support for configuration export in Drupal core. Now, thanks to the Configuration API built into Drupal 8, we're able to focus exclusively on the task of packaging up configuration--what Features is all about. The Feature module is not a dependency for sites that install feature modules.

> Automatic packaging. Rather than requiring you to manually create individual features, Features analyzes your site and automatically packages up site configuration into a set of features.